site/components/payment-streams/CreateStream.js

```diff
@@ -187,11 +187,16 @@ const CreateStream = function () {
         onYearsChange={setYears}
         years={years}
       />
-      <div className="flex">
-        <Link href="/payment-streams">
-          <Button className="w-19 m-1">{tCommon('cancel')}</Button>
-        </Link>
-        <Button className="w-19 m-1" disabled={!canSubmit}>
+      <div className="flex w-full space-x-4">
+        <Button
+          className="w-1/2"
+          onClick={function () {
+            router.push('/payment-streams')
+          }}
+        >
+          {tCommon('cancel')}
+        </Button>
```

Comment:
was this changed from a link into a button with `router.push` for any particular reason? `button` can be a child of `Link`

Reply:
In addition to that, given now this is a button, we need to be careful that it does not submit when its clicked, as it is inside a `form` component

Reply:
it was not styling correctly. So, In order to simplify this, I could have gone either way: style the link as a button and remove the button, or remove the link and leave the button with the onClick. For no particular reason, I chose the former. You're right though, it is submitting the form, It just happens that nothing gets done because its navigating to another page before, but to be sure I'll add a `type="button"`

Reply:
And no, <button> inside an <a> is not valid html